### ISSUE TITLE
Support Laravel 7

### ISSUE THREAD

**Issue by cnakh** [dugajean/laravel-repositories#15]
Laravel 7 is released.
Please make it support as well.

**Comment by Tjoosten** [comment]
There us an PR up check #14

**Comment by cnakh** [comment]
It's a nice and simple repository, but the owner seem ignored it.

**Comment by Tjoosten** [comment]
The issue seems to be resolved. This one can be closed.

**Comment by carlituxman** [comment]
with composer require dugajean/repositories:

ugajean/repositories 0.2 requires illuminate/support ^5.4|^6.0 -> found illuminate/support.....

**Comment by Tjoosten** [comment]
@carlituxman Can u explain a bit more? So i can write a fix for your issue.

**Comment by carlituxman** [comment]
Can't install in laravel 7. What version should I put in composer

**Comment by Tjoosten** [comment]
At first sight the tag is not up that includes the Laravel 7 support

**Comment by carlituxman** [comment]
Exactly, but in previous message say that includes a patch in the master
